.github/workflows/build-jsp.yml

```diff
@@ -73,6 +80,32 @@ jobs:
       with:
         name: UnicodeJsps
         path: UnicodeJsps/target/UnicodeJsps.war
-    - name: build docker image
-      run: cd UnicodeJsps && bash update-bidic-ucd.sh && docker build .
-
+    - name: Build C Bidi
```

Comment:
I think one step is missing, namely to get the Last Resort font:
https://github.com/unicode-org/unicodetools/blob/27875cb669668c0e42ea6866080f78c1566fe6e5/docs/unicodejsps/index.md#running-a-docker-based-build

Reply:
https://github.com/unicode-org/unicodetools/issues/1010.